bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/store/mem/impl/MemCachePersistenceObject.java

```diff
@@ -88,7 +91,15 @@ public String getContentType() {
      * @return Returned in <code>Map<String, List<String>></code> format.
      */
     public Map<String, List<String>> getHeaders() {
-        return (Map<String, List<String>>) (Map<?, ?>) Multimaps.asMap(headers);
+        Map<String, List<String>> map = new HashMap<>();
```

Comment:
Not sure if we need same keys; maybe this should be a multi map that has a List<String> and not a hashmap?